aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -192,7 +192,12 @@ module aptos_framework::genesis {
     /// If it exists, it just returns the signer.
     fun create_account(aptos_framework: &signer, account_address: address, balance: u64): signer {
         if (account::exists_at(account_address)) {
```

Comment:
this is a bit odd with both sides. I am not sure I like exists_at change, but ignoring that this should be 

`if (account::exists_at(account_address) && coin::is_account_registered<AptosCoin>(account_address)) {`


and then code is not repeated

Reply:
could you explain more here? I didn't quite understand what you mean here... why is coin involved?